webui/src/Buttons/EditButton/ActionsPanel.jsx

```diff
@@ -291,6 +293,39 @@ function ActionTableRow({ action, index, dragId, setValue, doDelete, doDelay, mo
 	})
 	preview(drop(ref))
 
+	useEffect(() => {
+		const actionSpec = actionsContext[action.label]
+		const options = actionSpec?.options ?? []
+
+		for (const option of options) {
+			if (typeof option.isVisibleFn === 'string') {
```

Comment:
Im not sure if this is the right place to do this, as we will be doing it many times over the lifetime of a browser tab. 
If we simply add a ` && typeof option.isVisible !== 'function'`, then that should avoid duplicating work unnecessarily.

Reply:
Got it. Fixing ...